spinalcordtoolbox/utils.py

```diff
@@ -149,6 +149,24 @@ def flush(self):
         self.fd2.flush()
 
 
+def abspath(fname):
+    """
+    Get absolute path of input file name or path. Deals with tilde.
+
+    '~/code/bla' ------------------> '/usr/bob/code/bla'
+    '~/code/bla/pouf.txt' ---------> '/usr/bob/code/bla/pouf.txt'
+    '/usr/bob/code/bla' -----------> '/usr/bob/code/bla'
+    '/usr/bob/code/bla/pouf.txt' --> '/usr/bob/code/bla/pouf.txt'
+
+    :param fname:
+    :return:
+    """
+    if fname is not '':
+        if fname[0] == '~':
+            fname = os.path.expanduser(fname)
```

Comment:
The check above here is unnecessary I think, `expanduser` returns the input unmodified if it is an empty string or doesn't begin with tilde.

Reply:
you're right! fixed in fab5fcc1114addc76c6e6594fdcf2526467722f2